.infrastructure/helm-chart/todoapp/charts/mysql/templates/secret.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-secrets
+  namespace: {{ .Values.mysql.namespace }}
+type: Opaque
+data:
+  {{- range $k, $v := .Values.mysql.secrets.data }}
+    {{ $k }}: {{ $v | quote }}
+  {{- end }}
```

Comment:
Fix EoF

Reply:
fixed in [fix EoF, add serviceAccount YAML file, use pv storage value from value.yml](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_helm_charts/pull/30/commits/72ec2793e112758cfac44d2779c6d423be281e55) commit